src/VisionModel/resnet.jl

```diff
@@ -0,0 +1,104 @@
+using Flux
+
+export ResNet18, ResNet34, ResNet50, ResNet101, ResNet152
+
+basicblock(inplanes, outplanes, downsample = false) = downsample ?
+  Chain(Conv((3, 3), inplanes => outplanes[1], stride = 2, pad = 1),
+        BatchNorm(outplanes[1], relu),
+        Conv((3, 3), outplanes[1] => outplanes[2], stride = 1, pad = 1),
+        BatchNorm(outplanes[2], relu)) :
+  Chain(Conv((3, 3), inplanes => outplanes[1], stride = 1, pad = 1),
+        BatchNorm(outplanes[1], relu),
+        Conv((3, 3), outplanes[1] => outplanes[2], stride = 1, pad = 1),
+        BatchNorm(outplanes[2], relu))
+
+bottleneck(inplanes, outplanes, downsample = false) = downsample ?
+  Chain(Conv((1, 1), inplanes => outplanes[1], stride = 2),
+        BatchNorm(outplanes[1], relu),
+        Conv((3, 3), outplanes[1] => outplanes[2], stride = 1, pad = 1),
+        BatchNorm(outplanes[2], relu),
+        Conv((1, 1), outplanes[2] => outplanes[3], stride = 1),
+        BatchNorm(outplanes[3], relu)) :
+  Chain(Conv((1, 1), inplanes => outplanes[1], stride = 1),
+        BatchNorm(outplanes[1], relu),
+        Conv((3, 3), outplanes[1] => outplanes[2], stride = 1, pad = 1),
+        BatchNorm(outplanes[2], relu),
+        Conv((1, 1), outplanes[2] => outplanes[3], stride = 1),
+        BatchNorm(outplanes[3], relu))
+
+function projection(inplanes, outplanes, downsample = false)
+  shortcut = downsample ? 
+    Chain(Conv((1, 1), inplanes => outplanes, stride = 2),
+          BatchNorm((outplanes), relu)) :
+    Chain(Conv((1, 1), inplanes => outplanes, stride = 1),
+          BatchNorm((outplanes), relu))
+  return (x, y) -> x + shortcut(y)
+end
+
+# array -> PaddedView(0, array, outplanes) for zero padding arrays
+function identity(inplanes, outplanes, stride)
```

Comment:
One last change. Can you remove `stride` from the arguments? Since `projection` and `identity` no longer need to have matching function signatures.

Reply:
yeah!

Reply:
I had to update the CI stuff anyways, so I took care of it since it was a just mechanical issue.